styles/_variables.scss

```diff
@@ -1,12 +1,6 @@
 // Variables
 
 // Colors
-$brand_color: #17494d !default;
-$brand_text_color: #fff !default; //Color for text with an accent color background
-$text_color: #333 !default;
-$link_color: #17494d !default;
-$background_color: #fff !default;
```

Comment:
Looks like we can't really remove those? At least not all of those, ie `$brand_color` is used in the same file, in the line `10`. Are those removals intentional?

Reply:
These are already defined in the manifest.json file. Once we compile the SCSS those variables become available